salt/pythontest/init.sls

```diff
@@ -4,6 +4,25 @@ include:
 git:
   pkg.installed
 
+ftp:
```

Comment:
Better yet I'll see if we can make the ftp_username `nobody`, that way its garaunteed to have the least amount of privileges, though I'm not sure how that'll fare considering it has to bind to port 21. I'll test and see if that works, if not, then yeah we can call it `testftp`